students/20th/wonyoungcho/westagram/js/main.js

```diff
@@ -0,0 +1,29 @@
+'user strict';
+
+const idName = 'wychrischo ';
+const comment = document.querySelector('.postUserContainer');
+const addButton = document.querySelector('.commentsPostButton');
+const textArea = document.querySelector('.commentsInputComment');
+
+function commentAdd () {
+  const newComment = textArea.value;
+  const newDiv = document.createElement('div');
+  newDiv.className = 'post-user';
+
+  comment.append(newDiv);
+  newDiv.innerHTML = `<div class = 'postUser'><span class = 'postUserId'>${idName}</span> ${newComment}</div>`;
+  textArea.value = ''; 
+  addButton.style.color = '#8abdff';
```

Comment:
```suggestion
  addButton.style.color = EMPTY_BLUE;
```
색상 같은 것들도 변수 선언해서 사용하셨듯이 그렇게 사용하셔도 좋을 것 같고, 상수 변수로 선언해서 사용해도 좋겠습니다.